packages/bolt-matrix/mod.ts

```diff
@@ -31,23 +31,6 @@ export default class MatrixPlugin extends BoltPlugin {
 			homeserverUrl: this.config.homeserverUrl,
 			domain: this.config.domain,
 			registration: this.config.reg_path,
-			bridgeEncryption: {
```

Comment:
Is there any reason to remove all the encryption code? I'd prefer it if bolt-matrix supported encrypted rooms but if there's a reason I guess that's fine

Reply:
because it doesn't launch on my end with it, i might revisit it in the future, but almost all matrix-appservice-${platform} don't have encryption

Reply:
ah okay, i guess we could go back to that in the future then